Epsilon.Abstractions/OutcomeGradeStatus.cs

```diff
@@ -1,5 +1,8 @@
+using System.Diagnostics.CodeAnalysis;
+
 namespace Epsilon.Abstractions;
 
+[SuppressMessage("ReSharper", "UnusedType.Global")]
```

Comment:
Could you add justification to this, smth like will be implemented in the near future? Just for clarity